portfolio/src/main/webapp/script.js

```diff
@@ -442,15 +456,72 @@ function displayDirectionsAPI() {
   }
 }
 
-/*
-function repeatMarkers() {
-    fetch('/markers').then(response => response.json()).then((markers) => {
-    markers.forEach((marker) => {
-        var myPosition = new google.maps.LatLng(marker.lat, marker.lng);
-        if (google.maps.geometry.poly.isLocationOnEdge(myPosition, routeArray, 0.0064)) {
-            
-        }
-        });
+function repeatedMarkersUI() {
+    console.log("HEYYYYY");
```

Comment:
remove. you shouldn't need to do this. using the console, you can set breakpoints and walk through js line by line  https://developers.google.com/web/tools/chrome-devtools/javascript